typesense-java/src/main/java/org/typesense/model/Collections.java

```diff
@@ -0,0 +1,23 @@
+package org.typesense.model;
+
+import org.typesense.api.CollectionResponse;
+import org.typesense.api.CollectionSchema;
+
+public class Collections {
+
+    private static Api api;
+    public final static String RESOURCE_PATH = "/collections/";
+
+    public Collections(Api api1){
```

Comment:
Can you get away with: Collections.api = api instead of api1? Not sure, just a minor OCD query :).

Reply:
Sure! Don't know how the '1' got into the picture. Will change that